### ISSUE TITLE
swaggerui collapsed view by default

### ISSUE THREAD

**Issue by JoelAdamWeiss** [domaindrivendev/Swashbuckle.AspNetCore#279]
I attempted to use:

app.UseSwaggerUi(c =>
{
    c.EnabledValidator(null);
    c.DocExpansion("NONE");
});

To set the desired behavior, but regardless of what I set c.DocExpansion to, it's expanding to show all of the endpoints for each of my APIs - wouldn't be too bad if i didn't have a TON.

Am I setting this to the wrong value?  Kind of took a mix of the ASP.NET.Core settings & the reference for the .Net version.  I'm just trying to achieve the old default view that the .Net version had.

Thanks again for any help.

**Comment by domaindrivendev** [comment]
"none" should work. Most of the SwaggerUi middleware options map pretty closely to the corresponding swagger-ui parameters which you can read about here - https://github.com/swagger-api/swagger-ui#swaggerui

**Comment by JoelAdamWeiss** [comment]
I’ve tried that before, and I just tried again.  Still full expansion of endpoints, not fully expanded, but showing all of the calls for each endpoint.

 

I also tried sending ONLY that option, but that had the same effect.  Is there anything I could do to assist in ‘troubleshooting’ / fixing the issue?

 

Thanks!

**Comment by coderabsolute** [comment]
The following code has solved the problem for me, using v3.0.0

```
app.UseSwaggerUI(c =>
{
  c.DocExpansion(DocExpansion.None);
});
```

**Comment by gsundarraj** [comment]
Thank you. It works
